### PR TITLE
Add Fish Fight and update name of rust-rpg-toolkit

### PR DESCRIPTION
**Pull request by olefasting**
This adds `Fish Fight` to the list of games. Wasn't sure whether or not to keep the `Fish Game` entry, so I just left it in. I guess it is a good example repository, on a much smaller scale than `Fish Fight`, even though it is no longer actively developed...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,4 +96,4 @@ Plugin has additional `.js` file, so to use it, you need to do extra work. Befor
 - [`gesture-recogrizer`](https://github.com/optozorax/gesture-recognizer) - library for recognizing simple touch gestures.
 - [`sapp-console-log`](https://github.com/canadaduane/sapp-console-log) - [`log-rs`](https://github.com/rust-lang/log) compatible logging for sapp-wasm.
 - [`quad-rand`](https://github.com/not-fl3/quad-rand) - wasm-friendly random number generator for quads.
-- [`circle2d`](https://github.com/koalefant/circle2d) - little physics library with only circles, used in [Crate Before Attack](https://cratebeforeattack.com).
+- [`circle2d`](https://github.com/koalefant/circle2d) - little physics library with only circles, used in [Crate Before Attack](https://cratebeforeattack.com).
```
Comment (ozkriff) [inline README.md:99]: :thinking: what was changed here?

**Comment by ozkriff** [comment]
Yeah, Fish Game has value on its own, especially considering the tutorial that covers it.

File: README.md (hunk 1 above)
Comment (olefasting) [reply README.md:99]: Must be a blank line that was removed. or something. I have a fork of macroquad-particles, with serde support, that we made for Fish Fight. I added an entry for that, right under circle2d's, but I removed it again as I decided to wait until we have some other stuff added to it.

**Review by ozkriff — APPROVED** [review]

**Comment by ozkriff** [comment]
Tnx!

**Pull request merged**